Install-Django-Flask.md

```diff
@@ -12,249 +12,17 @@ In case these assumptions are untrue, you might want to take a look at this [wik
 
 But it would be unfair if we completely ignore the [Python 2 vs Python 3](http://docs.python-guide.org/en/latest/starting/which-python/#the-state-of-python-2-vs-3) debate.
 
-## Python 2 vs. Python 3
+# Install Django Flask
```

Comment:
Sorry but I didn't notice there was a title already so there is no need to add this one.